src/linear_algebra/dual.lean

```diff
@@ -570,6 +583,24 @@ begin
     exact is_basis.to_dual_equiv _ hB.1 },
 end
 
+lemma dual_annihilator_comp_findim_eq {Φ : subspace K (module.dual K V)} :
```

Comment:
Typo: `comp` -> `comap`. Usually we put the name of the function (`findim`) before the argument `dual_annihilator_comap`. Finally, this looks like a good `@[simp]` lemma.
```suggestion
@[simp]
lemma findim_dual_annihilator_comap_eq {Φ : subspace K (module.dual K V)} :
```